README.md

```diff
@@ -7,39 +7,58 @@ The search engine takes text queries (exemples provided in the data folder) and
 
 ## Set up
 
-Download the dataset [here](http://web.stanford.edu/class/cs276/pa/pa1-data.zip).
+This project is written with **Python 3**. All the following commands using python are implicitly using Python 3.
+
+Download the full dataset [here](TODO-link-to-drive).
```

Comment:
Could someone upload the `data/` directory for me in a drive and put the link here ?